package.json

```diff
@@ -1,39 +1,56 @@
 {
-	"name": "iei-xlsx",
```

Comment:
most of this is npm formatting the file; more evidence that this lib was compiled _then_ added to source control.

This is all for a `npm install -S crypto` to remove some magic requires